extraction/content_extractors/image_extractor.py

```diff
@@ -34,6 +34,12 @@ def __create_image(self, node, caption_tag=None):
         else:
             image_url = node['src']
 
+        is_invalid, image_url \
+            = utils.invalid_url(image_url)
+
+        if is_invalid:
+            return None
```

Comment:
Would it be better to return a proper error here?

Reply:
This will reject a particular media url, we don't need to show the error to user for each invalid media url that we are rejecting.